calendar/page.js

```diff
@@ -395,9 +395,10 @@ async function configureGristSettings() {
 }
 
 // when a user selects a record in the table, we want to select it on the calendar
-function gristSelectedRecordChanged(record, mappings) {
+async function gristSelectedRecordChanged(record, mappings) {
```

Comment:
This method must be run after updateCalendar method, so we need to wait for the same promise to resolve